### DISCUSSION TITLE
有办法得到单个字符的识别框吗？

### DISCUSSION THREAD

**Discussion by 0-Maxwei-0** [PaddlePaddle/PaddleOCR#12943] [Q&A]
推理完成后只能得到整个字符串的识别框，有办法得到单个字符的识别框和识别结果吗？

**Comment by g39088902** [comment]
有办法，并且该功能已经完成，但该功能被作为商用功能集成在百度智能云OCR服务中。因此作为个人用户可以通过掏钱的方式解决问题。
